generate_replacements.py

```
@@ -0,0 +1,49 @@
+    "src/main/java/io/github/carlos_emr/carlos/casemgmt/service/CaseManagementManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/RxManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/PrescriptionManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/PreventionManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/PMmodule/service/AdmissionManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/DrugLookUpManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/services/security/SecurityManager.java"
+]
+
+for fpath in files:
+    try:
+        orig = subprocess.check_output(['git', 'show', f'origin/main:{fpath}']).decode('utf-8').splitlines()
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)